dashboard/src/actions/profileActions.js

```diff
@@ -15,18 +15,17 @@ export const getProfileDetails = () => async (dispatch) => {
         type: TYPES.GET_USER_DETAILS,
         payload: response?.data,
       });
-    } else {
-      let toast = {
-        variant: "danger",
-        title: response?.data?.message,
-      };
-      dispatch({
-        type: TYPES.SHOW_TOAST,
-        payload: { ...toast },
-      });
-    }
+    } 
```

Comment:
Are we sure we won't get a status of 200 or a response length of zero without an error being thrown?